crates/ink/Cargo.toml

```diff
@@ -28,6 +28,7 @@ scale-info = { workspace = true, default-features = false, features = ["derive"]
 derive_more = { workspace = true, features = ["from"] }
 const_format = { workspace = true }
 keccak-const = { workspace = true }
+linkme = { workspace = true }
```

Comment:
Just checking for my understanding as new to the codebase. The `ink_metadata` dep was optional as above. If `linkme` was only used by the metadata crate previously, should it not perhaps be optional here as well? I see that this was due to `ink_metadata/std`, but wonder if this will still result in an unnecessary build of the crate in no-std scenarios. Better to be explicit?

Usages seem to be feature gated by `std`

Reply:
Yes, it should be optional, thanks for catching this ... fixed! 🙂